volumina/imageView2D.py

```diff
@@ -166,6 +171,48 @@ def __init__(self, parent, cropModel, imagescene2d):
         # For screen recording BlankCursor doesn't work
         #self.hiddenCursor = QCursor(Qt.ArrowCursor)
 
+    def _onRotateLeft(self):
+        print "-----------------------------------------------------------------------------------------------------"
```

Comment:
Actually, it looks like this function and the two below are unused.  Just remove them entirely.